x-pack/plugins/ml/public/application/datavisualizer/index_based/components/search_panel/field_type_filter.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useMemo } from 'react';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { Option, MultiselectPicker } from '../../../../components/multi_select_picker';
+import { FieldTypeIcon } from '../../../../components/field_type_icon';
+
+export const ML_JOB_FIELD_TYPES = {
```

Comment:
Could the `ML_JOB_FIELD_TYPES` enum in `ml/common/types/field_types.ts` be converted to a `const` and used instead of defining it again here

Reply:
Updated here https://github.com/elastic/kibana/pull/85726/commits/05bb9f39066f6955cb56ab4f09cc4f793dfcdef4